Logo/bin/generate-assets.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+
+wd=`pwd`
+destdir="autogenerated"
+pngwidths="300 600 1200"
+totalassets=6
+
+find . -type f | grep svg >/dev/null || {
+	echo "No suitable files found. Sure we are in the right dir?"
+	return
+}
+
+# Dirty check for opened vectors
+grep -l -E ' d="[^"]*[^zZ]"' *.svg >"$destdir/.temp" &&  (
+	echo "This/those files seems to have open paths!";
+	cat  "$destdir/.temp"
+	echo
+)
+	
+# Loop through SVG files 
+count=0;
+titleerrors="";
+
+for file in *.svg ; do
+	let "count++"
+	asset="${file%.*}"
+	
+	# Check for valid SVG title
+	svgtitle=$(awk 'BEGIN{IGNORECASE=1;FS="<title>|</title>";RS=EOF} {print $2}' $file)
+	if [ "$svgtitle" != "$asset" ]; then
+		titleerrors="${titleerrors}Title mismatch in ${file}, should be ${asset}, not ${svgtitle}\n"
+	fi
+	
+	# Build PNG in their resolutions
+	for width in $pngwidths; do
+		tempfile="${wd}/${destdir}/${asset}-${width}.png"
+		inkout=$(inkscape "${wd}/${file}" --export-width=$width --export-png=$tempfile)
```

Comment:
This will eat up all of the inkscape output, which probably isn't what we want. You can add `| tee >&2` to make the output show again:

```sh
inkout=$(inkscape "${wd}/${file}" --export-width=$width --export-png=$tempfile | tee >&2)
```

Reply:
Really I wanted to eat up all!
It's information about what inkscape have done, no need to print on screen.
Or should it be printed?
```
Background RRGGBBAA: ffffff00
Area 0:0:594,721:115,322 exported to 1200 x 233 pixels (193,704 dpi)
Bitmap saved as: /Users/simone/Desktop/ClassicPress-Design/Logo/autogenerated/classicpress-logo-wordmark-white-on-transparent-1200.png
```

Reply:
This is partly personal preference and partly a "what if", see below comment regarding the image height ...